config/samples/kustomization.yaml

```diff
@@ -2,8 +2,13 @@
 resources:
 - metallb.yaml
 - metallb.io_v1beta1_bfdprofile.yaml
+- metallb.io_v1beta1_bgppeer.yaml
```

Comment:
why we need the beta1 here?

Reply:
because it's a crd handled by the operator and we want a sample (and operator-sdk would complain). In any case this is mirrored from u/s